src/table/utils/styling-utils.js

```diff
@@ -73,12 +72,14 @@ export function getHeaderStyle(layout, theme) {
   const headerStyle = getBaseStyling(header, 'header', theme);
   headerStyle.borderWidth = '1px 1px 1px 0px';
 
-  // When the table background color from the sense theme is transparent,
+  // To avoid seeing the table body through the table head:
+  // - When the table background color from the sense theme is transparent,
   // there is a header background color depending on the header font color
```

Comment:
when did we decide to do this? I don't remember

Reply:
Here https://github.com/qlik-oss/sn-table/pull/442/files#diff-03c15d1e38a398c39952d4fe961d27a310bf4d1959f3b86b92a0cd7d5721f846R74